src/data/array_types.js

```diff
@@ -854,6 +854,7 @@ export class CollisionBoxArray extends StructArrayLayout6i1ul2ui2i24 {
     /**
      * Return the CollisionBoxStruct at the given location in the array.
      * @param {number} index The index of the element.
+     * @private
```

Comment:
Is there a way to prevent redundantly specifying `@private` on every method if the parent class is marked `@private` anyway?

Reply:
Looks like this is not supported. From the JSDoc [docs](https://jsdoc.app/tags-private.html):
> The @private tag is not inherited by child members. For example, if the @private tag is added to a namespace, members of the namespace can still appear in the generated output; because the namespace is private, the members' namepath will not include the namespace.